libqtile/widget/systray.py

```diff
@@ -40,15 +40,16 @@
 
 class Icon(window._Window):
     _windowMask = EventMask.StructureNotify | \
+        EventMask.PropertyChange | \
         EventMask.Exposure
 
     def __init__(self, win, qtile, systray):
         window._Window.__init__(self, win, qtile)
         self.systray = systray
-        self.width = systray.icon_size
-        self.height = systray.icon_size
+        self.protocol_version = 1
```

Comment:
Yes, you're right. 
Moreover, the version sent on embedded notify should be the max version supported by embedder, not the one supported by the client that is being sent now.
I'll update it when I'm home.